projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 11.3-a.7 - 2022-08-17
+### Bug fixes
+- VideoPress: remove inclusion modules/videopress/utility-functions.php from Jetpack shortcodes module. [#25660]
```

Comment:
```suggestion
- VideoPress: remove inclusion of modules/videopress/utility-functions.php from Jetpack shortcodes module to prevent errors. [#25660]
```

Or just `remove modules/videopress....`?

Same for the readme.

Reply:
It is clearer with added words but then again, this is Bug fixes section so fixing errors is expected?